### DISCUSSION TITLE
Get mentioned data

### DISCUSSION THREAD

**Discussion by steelze** [ueberdosis/tiptap#870] [General Questions]
How do I get the data of suggestions that were selected by the user? I need to get the id of the user so I can notify them. Thanks

**Comment by hanspagel** [comment]
We do this in the backend. When the content is stored in the database, we loop through the JSON and collect user ids. After storing the content, an email is sent to those users.

Hope that helps!

**Comment by steelze** [comment]
Can you help point to the property that has the user ids? Thanks

On Wed, 21 Oct 2020 at 10:45, Hans Pagel <notifications@github.com> wrote:

> We do this in the backend. When the content is stored in the database, we
> loop through the JSON and collect user ids. After storing the content, an
> email is sent to those users.
>
> Hope that helps!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ueberdosis/tiptap/discussions/870#discussioncomment-107457>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJKXBRF5PG23RFBDKP7DANTSL2UULANCNFSM4SZMUAAA>
> .
>

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>
